.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
```

Comment:
Looks like this is due to `atom-package-deps` not being able to determine the package name automatically (see [here](https://travis-ci.org/jschroeder9000/linter-haml/jobs/115568631#L268)). It should work properly if the name is directly specified [here](https://github.com/AtomLinter/linter-haml/blob/v1.2.0/lib/linter.coffee#L16).